### PR TITLE
DTSPO-2610 Add information for LetsENcrypt Cert Renewals

### PR DESCRIPTION
**Pull request by zc-hmcts**
### JIRA link ###
https://tools.hmcts.net/jira/browse/DTSPO-2610

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```
Screenshot 2021-06-28 at 16 45 40
Screenshot 2021-06-28 at 16 52 39

### CODE REVIEW

File: source/information-security/certificate-automation.html.md.erb [hunk 1]
```diff
@@ -11,7 +11,7 @@ weight: 3
 
 Where technically suitable, all new HMCTS domains must use automated certificate techniques and services, 
 such as [Microsoft managed certificates](https://docs.microsoft.com/en-us/azure/frontdoor/front-door-custom-domain-https#option-1-default-use-a-certificate-managed-by-front-door)
-(most preferred) or LetsEncrypt (uses ACME)
```
Comment (timja) [inline source/information-security/certificate-automation.html.md.erb:14]: why did you remove this?

File: source/information-security/certificate-automation.html.md.erb [hunk 2]
```diff
@@ -23,6 +23,15 @@ unexpected certificate expiry.
 
 ## Manual certificate requests
 
+Where the use of LetsEncrypt is suitable, New requests for certificates should be submitted to the Platform Operations team by raising a DTSPO Jira ticket containing the following information:
```
Comment (timja) [inline source/information-security/certificate-automation.html.md.erb:26]: I think we should re-word this a bit.

If your site is being hosted on:
(Link the below services to their product docs)
 * Azure FrontDoor
 * Azure App Service
 * Azure CDN
Then you must use the 'Microsoft managed certificates'

Otherwise please raise a request in #platops-help (there should be a redirect link used for this)

Comment (zc-hmcts) [reply source/information-security/certificate-automation.html.md.erb:26]: Screenshot 2021-06-29 at 13 52 51

Comment (timja) [reply source/information-security/certificate-automation.html.md.erb:26]: better can you push it though. remove the Once complete raise a request in platopshelp bit.

remove the CSR form bit as that will move to ops-runbooks

Comment (zc-hmcts) [reply source/information-security/certificate-automation.html.md.erb]: Screenshot 2021-06-30 at 10 51 37
pushed + screenshot

File: source/information-security/certificate-automation.html.md.erb [hunk 3]
```diff
@@ -23,7 +23,17 @@ unexpected certificate expiry.
...
+Then you must use the [Microsoft managed certificates](https://docs.microsoft.com/en-us/azure/frontdoor/front-door-custom-domain-https#option-1-default-use-a-certificate-managed-by-front-door)
+Otherwise please raise a request in [#platops-help](https://app.slack.com/client/T1L0WSW9F/C8SR5CAMU),
+the ticket should contain the following information:
 
-To request a manually issued certificate, complete the [certificate request form](https://tools.hmcts.net/confluence/display/RD/Requesting+SSL+Certificates) and send it, 
-with a Certificate Signing Request (CSR), to Opsconfman@hmcts.net.
+- Where the certificate is to be hosted
+- DNS Name and SANS
+
+Where automated certificate renewal is not possible, new certificates must be acquired through the DTS Operations Management team.
```
Comment (timja) [inline source/information-security/certificate-automation.html.md.erb:39]: I don't think this adds anything
```suggestion
```

Comment (cakeben) [reply source/information-security/certificate-automation.html.md.erb:39]: how about "If your site isn't being hosted on Front Door, App Service or CDN go here - https://hmcts.github.io/information-security/certificate-automation.html#manual-certificate-requests" or something to that affect

Comment (timja) [reply source/information-security/certificate-automation.html.md.erb:39]: yeah should work, move the required section up above the heading and then make this section just about raising the request?

**Review by timja — APPROVED** [review]
